### ISSUE TITLE
Use of `aes(color=category)` in `geom_density` or `geom_hist`

### ISSUE THREAD

**Issue by dmbates** [TidierOrg/TidierPlots.jl#102]
**Describe how it works in R's `ggplot2`**

If `x` is a numeric variable and `g` is a factor in data.frame `df` in R then `ggplot(df, aes(x = x, color=g) + geom_density()` produces a plot with multiple kernel density estimates.  See the `diamonds` example in https://ggplot2.tidyverse.org/reference/geom_density.html

In `TidierPlots.jl` it seems that the `color/colour` aesthetic is interpreted as a scalar color by geom_density.

Is there a way to create a plot with multiple kernel density curves in `TidierPlots.jl`?

BTW, @rdboyes It is great to see someone from the Kingston area (are you on Wolfe Island?) in Julia development.  I did both my undergraduate degree and my Ph.D. at Queens.

**Comment by rdboyes** [comment]
Yep! Wolfe Island - just finished my Ph.D at Queen's last year.

Thanks for reporting this - it's another issue downstream of the grouping issue (like #93) - looks like @adknudson has an idea how to fix it over there so we should have this working soon!

**Comment by rdboyes** [comment]
In dev branch now: 

```julia
ggplot(penguins) +
         geom_density(aes(x = :bill_length_mm, color = :island))
```

![image](https://github.com/TidierOrg/TidierPlots.jl/assets/33524191/febe6f33-8668-47ce-a86b-ac7fc2073af1)

**Comment by rdboyes** [comment]
Interesting - it seems to be plotting all three curves independently, but all in blue. I'll look into this change - it could be due to the new version of Makie? But I'm not sure yet

**Comment by rdboyes** [comment]
This is "re-fixed" in 0.8.0 (to be released soon)
